contracts/src/GrandsumVerifier.sol

```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+contract GrandsumVerifier {
+    // Calldata positions for proofs
+    uint256 internal constant       PROOF_LEN_CPTR = 0x64;
+    uint256 internal constant           PROOF_CPTR = 0x84;
+
+    // Memory positions for the verifying key.
+    // The memory location starts at 0x200 due to the maximum operation on the ec_pairing function being 0x180, marking the maximum memory location used
+    uint256 internal constant                VK_MPTR = 0x200;
+    uint256 internal constant         VK_DIGEST_MPTR = 0x200;
```

Comment:
VK_DIGEST_MPTR is not used

Reply:
True, I will address this by copying only the values needed in the verifier from the VerifyingKey.